php/libraries/Utility.class.inc

```diff
@@ -645,33 +618,6 @@ class Utility
         $stage = $db->pselect($query, array('CandID' => $Cand_id));
         return intval($stage[0]['SubprojectID']);
     }
-
```

Comment:
Thia function seems to just be moved in the same class not sure what the changes are